data/json/npcs/isolated_road/isolated_road_cody_fabricate.json

```diff
@@ -46,23 +46,23 @@
       "type": "dialogue",
       "context": "artisans",
       "value": "yes",
-      "yes": "Alright <u_val:number_artisans_blacksmith_thickness>mm, so the next question is time.  Currently, it's the classic problem, if you want quality, you gotta wait.  Two processes I could offer that you would be hard-pressed to find anywhere else are either really great high carbon steel or, I can go through the lengthy process of tempering.  High carbon will take three weeks to process and do the work, and tempering will take me four.  For five I could, I think, create a suit of that Nomad armor.  It would incorporate climate control, help you support more weight, and it'll have a jumpsuit built in but you will not be able to wear normal clothes with it.",
-      "no": "Alright <u_val:number_artisans_blacksmith_thickness>mm, so the next question is time.  Currently, it's the classic problem, if you want quality, you gotta wait.  Two processes I could offer that you would be hard-pressed to find anywhere else are either really great high carbon steel or, I can go through the lengthy process of tempering.  High carbon will take three weeks to process and do the work, and tempering will take me four."
+      "yes": "Alright.  <u_val:number_artisans_blacksmith_thickness>mm.  so the next question is time.  Currently, it's the classic problem: if you want quality, you gotta wait.  Two processes I could offer that you would be hard-pressed to find anywhere else are either really great high carbon steel, or I can go through the lengthy process of tempering.  High carbon will take three weeks to process and do the work, and tempering will take me four.  For five I could, I think, create a suit of that nomad armor.  It would incorporate climate control, help you support more weight, and it'll have a jumpsuit built in, but you won't be able to wear normal clothes with it.",
```

Comment:
should the first bit be three sentences?

"Alright. 2-4-6mm. so the next question is time" doesn't look right to me.